LICENSE.md

```diff
@@ -0,0 +1,9 @@
+# The MIT License (MIT)
+
+Copyright (c) `2016` `CloudBees, Inc.`
```

Comment:
Lets move LICENSE changes to another PR so I can get feedback from @larrys (it's his IP)